### ISSUE TITLE
Anti-regulator of G-protein signaling 8 (RGS8) antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10546]
**Preferred term label:**

**Synonyms**
Anti-RGS8

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against regulator of G-protein signaling 8 (RGS8)

PMID: 33782191 This study provided Class IV evidence that autoantibodies against the intracellular Purkinje cell protein RGS8 are associated with paraneoplastic cerebellar syndrome in lymphoma. 

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624259 Non-specific autoimmune cerebellar ataxia with characteristic antibodies

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
